src/System.Runtime.Intrinsics.Experimental/ref/System.Runtime.Intrinsics.cs

```diff
@@ -25,6 +25,148 @@ public struct Vector256<T> where T : struct
+        public static Vector64<uint>    Multiply(Vector64<uint>    left, Vector64<uint>    right) { throw null; }
+        public static Vector64<int>     Multiply(Vector64<int>     left, Vector64<int>     right) { throw null; }
+        public static Vector64<float>   Multiply(Vector64<float>   left, Vector64<float>   right) { throw null; }
+        public static Vector128<byte>   Multiply(Vector128<byte>   left, Vector128<byte>   right) { throw null; }
+        public static Vector128<sbyte>  Multiply(Vector128<sbyte>  left, Vector128<sbyte>  right) { throw null; }
+        public static Vector128<ushort> Multiply(Vector128<ushort> left, Vector128<ushort> right) { throw null; }
+        public static Vector128<short>  Multiply(Vector128<short>  left, Vector128<short>  right) { throw null; }
+        public static Vector128<uint>   Multiply(Vector128<uint>   left, Vector128<uint>   right) { throw null; }
+        public static Vector128<int>    Multiply(Vector128<int>    left, Vector128<int>    right) { throw null; }
+        public static Vector128<float>  Multiply(Vector128<float>  left, Vector128<float>  right) { throw null; }
+        public static Vector128<double> Multiply(Vector128<double> left, Vector128<double> right) { throw null; }
+        public static Vector64<sbyte>   Negate(Vector64<sbyte>   value) { throw null; }
+        public static Vector64<short>   Negate(Vector64<short>   value) { throw null; }
+        public static Vector64<int>     Negate(Vector64<int>     value) { throw null; }
+        public static Vector64<float>   Negate(Vector64<float>   value) { throw null; }
+        public static Vector128<sbyte>  Negate(Vector128<sbyte>  value) { throw null; }
+        public static Vector128<short>  Negate(Vector128<short>  value) { throw null; }
+        public static Vector128<int>    Negate(Vector128<int>    value) { throw null; }
+        public static Vector128<long>   Negate(Vector128<long>   value) { throw null; }
+        public static Vector128<float>  Negate(Vector128<float>  value) { throw null; }
+        public static Vector128<double> Negate(Vector128<double> value) { throw null; }
+        public static Vector64<T>  Not<T>(Vector64<T>  value) where T : struct { throw null; }
+        public static Vector128<T> Not<T>(Vector128<T> value) where T : struct { throw null; }
+        public static Vector64<T>  Or<T>(Vector64<T>  left, Vector64<T>  right) where T : struct { throw null; }
+        public static Vector128<T> Or<T>(Vector128<T> left, Vector128<T> right) where T : struct { throw null; }
+        public static Vector64<T>  OrNot<T>(Vector64<T>  left, Vector64<T>  right) where T : struct { throw null; }
+        public static Vector128<T> OrNot<T>(Vector128<T> left, Vector128<T> right) where T : struct { throw null; }
+        public static Vector64<byte>    PopCount(Vector64<byte>    value) { throw null; }
```

Comment:
Does this instruction really only support bytes and not `uint` or `ulong`?

Reply:
Unfortunately, `cnt` is `Population Count per byte`

The wider forms can be synthesized in C# when we add `AddPairwise()`, `StaticCast()` and optionally `WidenLo()`

i.e.
```
public Vector64<ushort> PopCount(Vector64<ushort> value)
{
  Vector64<byte> popCountPerByte = PopCount(StaticCast<byte>(value));
  Vector64<byte> popCountPerElement = AddPairwise(popCountPerByte, SetAllVector64(0));
  return WidenLo(popCountPerElement);
}
public Vector64<uint> PopCount(Vector64<uint> value)
{
  Vector64<byte> popCountPerByte = PopCount(StaticCast<ushort>(value));
  Vector64<byte> popCountPerShort = AddPairwise(popCountPerByte, SetAllVector64(0));
  Vector64<byte> popCountPerElement = AddPairwise(popCountPerShort, SetAllVector64(0));
  return WidenLo(popCountPerElement);
}
```

Reply:
We could eventually add (or revise) to `vector64<byte> PopCountPerByte<T>(vector64<T> value)`;
It would eliminate the need for the static casts.